include/nigiri/loader/permutate_locations.h

```diff
@@ -11,7 +13,7 @@ namespace nigiri {
 static vector<location_idx_t> permutation_;
 static vector<location_idx_t> mapping_vec;
 
-vector<location_idx_t> create_mapping_vec(vector<location_idx_t> vec) 
+static vector<location_idx_t> create_mapping_vec(vector<location_idx_t> vec) 
```

Comment:
move to .cc file

(if in header, then use `inline` instead of `static` but if not a template and not performance sensitive -> move to .cc file)